mecademic_description/urdf/meca_500_r3_macro.xacro

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0"?>
```

Comment:
I find it a bit weird that this has the word `marco` in its filename. I think the convention is that all the files that end on `.xacro` are macros and only the `.urdf.xacro` files could actually generate a urdf.